### PR TITLE
Innitial code drop

### PR DESCRIPTION
**Pull request by PetkoNosal**

### CODE REVIEW

**Review by matusbabinsky — COMMENTED** [review]
intelij-plugin/src/main/gen/tech/pantheon/yanginator/plugin/lexer/jflex-1.7.0-2.jar

should not be versioned

**Comment by PetkoNosal** [comment]
> intelij-plugin/src/main/gen/tech/pantheon/yanginator/plugin/lexer/jflex-1.7.0-2.jar
> 
> should not be versioned

fixed

File: intelij-plugin/src/main/highlighter/tech/pantheon/yanginator/plugin/YangHighlighter.bnf [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      H_SYSTEM_KEYWORD      = 'system'
+      H_TRUE_KEYWORD        = 'true'
+      H_UNBOUNDED_KEYWORD   = 'unbounded'
+      H_USER_KEYWORD        = 'user'
+
+      H_WHITE_SPACE = 'regexp:\s+'
+      H_STRING = "regexp:\"([^\"])*\""
+      H_DATE_ARG='regexp:[0-9]{4}-[0-9]{1,2}-[0-9]{1,2}'
+      H_IDENTIFIER = 'regexp:[a-zA-Z_][0-9a-zA-Z_\-.]*'
+      H_EXTENSION_IDENTIFIER = 'regexp:[a-zA-Z_][0-9a-zA-Z_\-.:]*'
```
Comment (rovarga) [inline intelij-plugin/src/main/highlighter/tech/pantheon/yanginator/plugin/YangHighlighter.bnf:115]: Does this allow for "extension foo:bar:baz"?

Comment (matusbabinsky) [reply intelij-plugin/src/main/highlighter/tech/pantheon/yanginator/plugin/YangHighlighter.bnf:115]: this is highlighter's grammar this does

**Pull request merged**